app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -472,6 +484,7 @@ const CallScreen = ({
     const waitingForRecording = Boolean(currentCall?.recState?.init_at && !currentCall.recState.start_at && !currentCall.recState.end_at && isHost);
     const showStartRecording = isHost && EnableRecordings && !(waitingForRecording || recording);
     const showStopRecording = isHost && EnableRecordings && (waitingForRecording || recording);
+    const ccAvailable = Boolean(EnableRecordings && EnableTranscriptions && EnableLiveCaptions && recording);
```

Comment:
More of a comment than a request but I feel this is a lot and if we ever decoupled the job it will essentially break. Wondering if we should instead have a single server side boolean to inform whether a live caption is available for the current call.

Reply:
Yah, could be another way to go. Of course it wouldn't simplify the logic, it would just move the logic to the backend--but then we wouldn't need to change the client in that future case. Is it on the horizon that we could have captions without transcriptions, and/or transcriptions without recordings?

Reply:
> Is it on the horizon that we could have captions without transcriptions, and/or transcriptions without recordings?

I am not sure to be honest but logic on the backend is advantageous because it can also be dynamic. For example, this boolean could go in the state call itself and potentially be turned to false in case of issues with live captions (e.g. due to performance) as we discussed.

Reply:
The changes are made in https://github.com/mattermost/mattermost-plugin-calls/pull/633 now, so adding them here.